virtool/dev/fake.py

```diff
@@ -177,29 +175,6 @@ async def create_fake_analysis(app: App):
     logger.debug("Created fake analyses")
 
 
-async def create_sample_integration_test_job(app: App):
-    name = "integration_test_workflow"
-    rights = JobRights()
-
-    return await virtool.jobs.db.create(
-        db=app["db"],
-        workflow_name=name,
-        job_args={
-            "sample_id": SAMPLE_ID_PAIRED_FINALIZED,
-            "subtraction_id": "subtraction_1",
-            "ref_id": "reference_1",
-            "analysis_id": "analysis_1"
-        },
-        user_id=USER_ID,
-        rights=rights,
-        job_id="sample_integration_test_job",
-    )
-
-
-async def create_fake_jobs(app: App):
-    await create_sample_integration_test_job(app)
-
-
```

Comment:
Can this be removed @BlakeASmith?

Reply:
Yes. The test job is now created in `virtool.fake.test_cases.sample_integration`